readme.md

```diff
@@ -42,3 +42,14 @@ should be ready to go.
 
 **It is CRITICAL that whole `app/`, `log/` and `temp/` directories are not accessible directly
 via a web browser. See [security warning](https://nette.org/security-warning).**
+
+Notice: Composer PHP version
+----------------------------
+This project forces `PHP 5.6` as your PHP version for Composer packages. If you have different version on production you should change it in `composer.json`.
+```json
```

Comment:
Because when I have old php version (and old web-project), then look on current web-project readme I see that only when I have newer version I should change the platform.
eg 5.4 < 5.6 => do nothing :-)